lib/ViewModels/TerriaViewer.js

```diff
@@ -315,6 +315,12 @@ us via email at '+that.terria.supportEmail+'.'
 
     var scene = viewer.scene;
 
+    // Set minimum zoom distance to 25 meters. Default is 1m.
+    // Zoom crashes through terrain if the rate of zoom is higher than terrain
+    // resolution rate and the height at the more detailed LOD is higher than
+    // previous LOD.
+    scene.screenSpaceCameraController.minimumZoomDistance = 25.0;
```

Comment:
Yeah I see that when I don't scroll fast enough. Magical teleportation can be accessed if you scroll faster still and keep scrolling. Do you want me to remove this threshold? FlyTo problem is catered for by changed to ZoomControl.jsx alone.